scripts/binary_size_check.sh

```diff
@@ -8,7 +8,8 @@ fi
 
 . ./scripts/version.sh
 
-MAX_BINARY_SIZE=61000000
+# Try to keep the K3s binary under 60 megabytes
+MAX_BINARY_SIZE=$((60 * 1024 * 1024))
```

Comment:
Can we not just fill this in as a constant?

Reply:
A constant where? The only place we use it is in this script to fail CI if we've bloated things too much.

Reply:
I mean fill it in statically - i.e. set to 62914560 instead of evaluating it every time the script is run. This is a nit though.

Reply:
Ah. I figured it was easier to read as a human being in this format; I don't off the top of my head know how many MiB 62914560 bytes is.

Reply:
`k3s binary dist/artifacts/k3s size 63610880 exceeds max acceptable size of 62914560 bytes`

Maybe time to let bloat set this to `64 MB`? 🤣

Reply:
It definitely is more readable, but IMO it could be a comment.

Either way, it doesn't block anything I was just looking to see if that is something we might want to do.

Regardless, I think the value needs to be bumped anyway because CI appears to be failing due to it being too low.

Reply:
CI should pass now that I've rebased on top of the k3s-root update. I also bumped it to 64MB for additional headroom.